plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -715,6 +715,14 @@ func NewServiceAffinityPredicate(podLister algorithm.PodLister, serviceLister al
 // the same service are running on nodes with
 // the exact same ServiceAffinity.label values).
 //
+// For example:
+//
+// If the first pod of a service was scheduled to a node with label "region=foo", all the other subsequent pods
```

Comment:
Can you please shorten the lines so that they fit into lines in github review tool (i.e. make them similar length to the above comment)?